app/team.go

```diff
@@ -196,7 +207,18 @@ func (a *App) createOnboardingLinkedBoard(c request.CTX, teamId string) (*fb_mod
 	const preferenceName = "linked_board_created"
 	userId := c.Session().UserId
 
-	boardService := a.Srv().services[product.BoardsKey].(product.BoardsService)
+	boardServiceItf, ok := a.Srv().services[product.BoardsKey]
+	if !ok {
+		// here make sure to return error when necessary, othewise just log the errors
+		return nil, model.NewAppError("CreateBoard", "not product key found", nil, "", http.StatusBadRequest)
```

Comment:
The second argument of `model.NewAppError` must be a "translatable" key like `app.team.create_onboarding_linked_board.board_not_product`. Then you can use `make i18n-extract` to pull the strings out.